### ISSUE TITLE
Javascript files not found in gulp-starter demo

### ISSUE THREAD

**Issue by HildoBijl** [vigetlabs/blendid#342]
I set up the gulp-starter demo (trough `git checkout demo`) and ran it through `npm run gulp`. The page mostly loaded OK, except the console indicated that various Javascript files (`shared.js` and `page.js`) could not be found.

Some research showed that various javascript scripts are loaded directly in the HTML code of the pages, like
`<script src="javascripts/shared.js"></script>`
The javascript files are subsequently not included in the build and hence fail to load. Is this a bug? Or should I enable this through some configuration settings?

**Comment by HildoBijl** [comment]
I found the "bug". Instead of checking out the `demo` branch, I checked out the `master` branch, then went to the `extras/demo/` folder, copied the `config.json` file and the `src` directory to the right places, and ran everything again through `npm run gulp`. This time nearly everything worked well.

Okay, I did get one error message in the console. The script could not find `sprites.svg`. Opening up the `extras/demo/src/html/macros/helper.html` file and replacing "sprites.svg" by "icons.svg" fixed this. Then everything was working properly. This is a minor fix, which I would suggest implementing.

But the main problem is that the `demo` branch is severely outdated. Many fixes have been applied to the `master` branch, especially in the `gulpfile.js` directory, which have not been applied to the `demo` branch. I would suggest using the `master` branch to set up another demo, and just use this to overwrite the `demo` branch entirely, since it is not functioning properly.
